src/pages/docs/providers/gpu/gpu-golem-live.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Installation instructions - gpu-golem-live
+hide:
+  - navigation
+description: gpu-golem-live project documentation
+type: Instructions
+---
+
+# ⚙️ Installation instructions - gpu-golem-live
+
+You are invited to join the `gpu-golem-live` project, an initiative to make GPU hardware resources available within the Golem Network. GPU resources will contribute to perform artificial intelligence (AI) tasks and more . As part of your participation in the network, you will receive GLM tokens as compensation for the computing services you provide.
+
+## Prerequisites
+
+To properly use the service as a provider in `gpu-golem-live`, it is necessary to meet the following hardware requirements:
+
+- A computer equipped with a motherboard and CPU with support for IOMMU (Input-Output Memory Management Unit) virtualization.
+- Nvidia graphics card at least series 30xx with 8 GB VRAM.
+- RAM - at least the same amount as VRAM, 2x more recommended.
+- SSD with a minimum capacity of 64 GB, connected via fast USB (USB 3.2 recommended). Alternatively, you can use a high-speed USB Flash Drive (although this is not recommended).
+- A stable Internet connection with a minimum of 100 Mbps. (for both downloading and uploading data), via an RJ45-type network cable (Ethernet).
+- Own ETH wallet
+
+By joining `gpu-golem-live`, you support AI infrastructure and manage your hardware resources flexibly, while getting the benefits of GLM tokens. If you have all of the above, you are welcome to join the project and work together for the development of artificial intelligence.
+
+Learn more at:
+[➜ golem-gpu-live @ GitHub](https://github.com/golemfactory/golem-gpu-live)
+
+## 1. Download and unzip image
+
+In the first step, download and unzip the `gpu-golem-live` provider image.
+
+To do this, click on the link below or copy the address and paste it in your web browser. The download will begin.
+
+[Download image](https://mirror.notset.fr/golem/golem-gpu-live-231215T091610Z.img.xz)
+
+```bash
+https://gpu-provider.dev.golem.network/golem-gpu-live-latest.img.xz
+```
+
+Alternatively, you can download the file using a terminal by typing the command:
+
+```bash
+wget https://gpu-provider.dev.golem.network/golem-gpu-live-latest.img.xz
+```
+
+The file takes about 2 GB and should download in about 3 minutes (on a 100 Mbps connection).
+
+Once the download is complete, unzip the file with any tool (such as the default archiving tool, The Unarchiver or 7-Zip) or use the command prompt:
+
+```
+gunzip golem-gpu-live-latest.img.xz
+```
+
+Done! The unzipped file should take up about 8.6 GB.
+
+## 2. Record image to SSD
+
+In the second step, you need to burn the unzipped `gpu-golem-live` image to an external SSD. To do this, you can use free software (such as balenaEtcher) or a command in the terminal.
```

Comment:
"burn" feels very old school, and might make the providers as seen on the discord channel worry that this is outdated tech. Use the term write or flash

Reply:
you're right, "flash" will be ok